src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_connection/manage_direct_query_data_connections_table.tsx

```diff
@@ -283,6 +386,7 @@ export const ManageDirectQueryDataConnectionsTable: React.FC<ManageDirectQueryDa
         </EuiFlexItem>
       </EuiFlexGroup>
       {isModalVisible && modalLayout}
+      {showIntegrationsFlyout && integrationsFlyout}
```

Comment:
Why not simply do this? But `setIntegrationsFlyout` to set the component to state?
```typescript
{showIntegrationsFlyout && <InstallIntegrationFlyout {...} />}
```